### ISSUE TITLE
Request for grade

### ISSUE THREAD

**Issue by AlexanderTahtadjian** [AlexanderTahtadjian/saas-website#6]
@muxlowc
I learnt a lot of stuff this semester about building a website, primarily just how helpful Typografier, Gridifier and Modulifer are very helpful and useful tools, I learnt how to code tables (which I included in the Products page, I loved learning how to create very large text after the "just your type" assignment as well as and accessibility for all types of users. This website proves that i understand a bit more advanced web building code from last semester along with new patterns that will help me a great deal in the future, I was also able to solve a lot more of my problems alone due to the fact that I understood a bit better what the lessons were giving me and I believe that I rose up to the challenge and did my best.

In terms of grade, I would give myself either a **B** or **C**. This is because even though it looks well done  as well as having that professional feel to it, I still feel like their could halve been a bit more room for improvement. for example, I feel like the testimonials could have turned out a lot better by maybe trying different classes to make them more centered or the fact that I could have been a bit more creative with the footer. I also could have tried a bit harder with the Accessibility& Functionality a bit more since I really only did the basic s for each since it's still relatively new to me. Nevertheless, I did the best I could with the knowledge and skills I had and that's all anybody could ask.

Some of the things that I struggled with this semester was mainly matching my code to the screenshots since even though my code would match the answers; it would still feel a bit off, but I still manged to solve the problem quickly (with a bit of help from my professor and peers). Another problem that I overcame was writing less code but still able to create great content since i sometimes tend to write too much code and end up overriding previous work.

In conclusion, the website shows I’ve learned a lot more advanced coding topics like accessibility, functionality and units&grids and I was able to  apply everything from the term to the best of my skills and I will not forget them anytime soon since I now have a better understanding on how coding now works and I can't wait for what will happen next year.
![Annotation 2020-04-23 224534](https://user-images.githubusercontent.com/55193211/80169739-45cea700-85b4-11ea-9910-c6fbcdd70484.jpg)
